components/script/dom/bindings/htmlconstructor.rs

```diff
@@ -377,10 +377,10 @@ pub fn push_new_element_queue() {
     ScriptThread::push_new_element_queue();
 }
 
-pub(crate) unsafe fn call_html_constructor<T: DerivedFrom<Element> + DomObject>(
+pub unsafe fn call_html_constructor<T: DerivedFrom<Element> + DomObject>(
```

Comment:
It's weird that `pub(crate)` isn't enough.

Reply:
I guess it's because `import.rs` tries to export it pub.

Reply:
Ah, makes sense. Although most stuff ins script create should be `pub(crate)` only (to get better dead code analysis) but we should do this after we resolve #1799.